app/components/results-list/component.js

```diff
@@ -70,7 +74,7 @@ export default Ember.Component.extend({
                 page = 0;
             }
             queryParams["page"] = page
-            this.attrs.transitionToFacet("resultsList", queryParams);
+            this.attrs.transitionToFacet("search", queryParams);
```

Comment:
What was the issue with pagination? It transitions to the search results page on production.